packages/core/admin/_internal/node/createBuildContext.ts

```diff
@@ -47,6 +47,10 @@ interface BuildContext {
    * The environment variables to be included in the JS bundle
    */
   env: Record<string, string>;
+  /**
+   * Features object with future flags
+   */
+  features?: FeaturesService['config'];
```

Comment:
What would you suggest then in this case?

It matches the config name which I think is correct to do because we're then able to manage features beyond future. But Im not precious and open to other ideas

Reply:
Let's go with what we have for now. It might change for v5 I think we need to unify multiple things more clearly